local-cli/generator/templates/index.android.js

```diff
@@ -28,9 +27,9 @@ var <%= name %> = React.createClass({
       </View>
     );
   }
-});
+}
 
-var styles = StyleSheet.create({
+const styles = StyleSheet.create({
```

Comment:
@Purii I think it's working because babel transpiles it and changes it to `var`. AFAIK, it will work in loose mode, but will throw a `ReferenceError` in strict mode.

So I think we should move it to top;